### ISSUE TITLE
Hi, how can I code to have a 5 minutes countdow everytime I come into the page?

### ISSUE THREAD

**Issue by OverwhelmingTOD** [rafaelveiga/obsidian-widgets#23]
I have only a little coding knowledge( only good at sql).
I want to achieve the function where a 5 minutes countdown will be reset everytime I come in the page.

I tried like this but it didnt work

```widgets
type: countdown
date: date("YYYY-MM-DD HH:MM:SS",time())
to: date("YYYY-MM-DD HH:MM:SS",time()+300)
``

Thx 
A lot Love

**Comment by PrabalSaxena01** [comment]
I think a template can help you with your problem

**Comment by rafaelveiga** [comment]
Hey @OverwhelmingTOD, we have a branch in the works #28 that solves this issue. This should be included in release 0.0.9. I'll post on this issue once it's released.

**Comment by rafaelveiga** [comment]
Hey @OverwhelmingTOD, we've just released 0.0.9. This should include this feature. Just add `+5m` to add a countdown of 5 minutes into a note
